### PR TITLE
Feature data migration redshift snowflake

### PR DESCRIPTION
**Pull request by nbanga01**
Feature: Historical ADF Pipeline for Redshift to Snowflake

## Description & motivation
Initial Commit from feature to main branch.

## To-do before merge
Wait till 5th Dec Data load, to do merge

## Screenshots:
NA

## Validation of ADF Artifacts:
- Data from Redshift to move in Snowflake.
- Check Metadata log tables for more status.

## Changes to existing ADF Artifacts:
No changes. It is a new development.

## Checklist:

- [x] I confirmed and have no doubts that there are no confidential data, sensitive details and credentials in my code.
- [x] My pull request represents one logical piece of work.
- [x] My commits are related to the pull request and look clean.
- [ ] I have added appropriate tests and documentation to any new ADF artifacts.
- [ ] I have updated the README file.

### CODE REVIEW

**Comment by nbanga01** [comment]
> hi @nbanga01 - your branch is out-of-date with the base branch. could you update it from main. Do a check with Akhil if in doubt. Do check with Akshay on the naming conventions to follow as well.

this is done!

File: pipeline/PIPE_hist_data_migration_redshift_snowflake_child.json
Comment (whow-kenvue-com) [inline pipeline/PIPE_hist_data_migration_redshift_snowflake_child.json:1]: this is still referencing SBX.. if not required, please remove.

**Review by whow-kenvue-com — CHANGES_REQUESTED** [review]
hi, pipeline/PIPE_hist_data_migration_redshift_snowflake_child is still having reference to SBX.. please remove if not required.

File: pipeline/PIPE_hist_data_migration_redshift_snowflake_child.json
Comment (nbanga01) [reply pipeline/PIPE_hist_data_migration_redshift_snowflake_child.json:1]: yes, removed!

**Review by whow-kenvue-com — APPROVED** [review]
ok.

**Review by awahi01 — APPROVED** [review]
QA team tested the data and confirmed everything looks fine, approving the PR.

**Comment by SKuma016** [comment]
Approved

**Review by SKuma016 — APPROVED** [review]
Approved

**Review by kkr01ken — APPROVED** [review]
We have validated 330 tables and the data was matching

**Comment by Murali-MK** [comment]
approved

**Review by Murali-MK — APPROVED** [review]
approved

**Review by whow-kenvue-com — APPROVED** [review]
enable auto-merge

**Pull request merged**